src/app/store/modules/river-linker.js

```diff
@@ -0,0 +1,58 @@
+import actions from "@/app/store/actions";
+import mutations from "@/app/store/mutations";
+import { getReachDocuments } from '@/app/services'
+
+export default {
+  namespaced: true,
+  state: {
+    error: false,
+    loading: false,
+    data: null,
+  },
+  mutations: {
+    ...mutations,
+
+  },
+  actions: {
+    ...actions,
+    async getProperty(context, id) {
+      try {
+        context.commit("DATA_REQUEST");
+
+        const result = await getReachDocuments(id);
```

Comment:
this file should probably just be `river-documents.js`, not linker, since it's getProperty is specific to documents, right? (more in line with the other changes you've made as well)

Reply:
though in that case I would put the `documents` getter logic directly into `getProperty` and just use `riverDocuments.data` rather than calling the documents getter